client/state/form/shipping-label/actions.js

```diff
@@ -1,9 +1,13 @@
+import * as FormActions from 'state/form/actions';
+
 export const OPEN_PRINTING_FLOW = 'OPEN_PRINTING_FLOW';
 export const EXIT_PRINTING_FLOW = 'EXIT_PRINTING_FLOW';
 
-export const openPrintingFlow = () => ( {
-	type: OPEN_PRINTING_FLOW,
-} );
+export const openPrintingFlow = () => ( dispatch ) => {
+	dispatch( { type: OPEN_PRINTING_FLOW } );
+	dispatch( FormActions.setFormProperty( 'pristine', false ) );
+	dispatch( FormActions.nextStep() );
+};
```

Comment:
I don't know if this kind of code is what we want to achieve. `redux-thunk` allows this kind of things to be done inside actions, so actions stop being plain objects and start becoming "recipes", or even places to put complex logic. What do you think about this vs having several reducers interpret the `OPEN_PRINTING_FLOW` action?